projects/demo-integrations/cypress/support/helpers/get-by-example-id.ts

```diff
@@ -0,0 +1,11 @@
+export function tuiEnsureExampleId(): string {
+    return Cypress.env(`EXAMPLE_ID`) ?? `tui-doc-example`;
+}
+
+export function tuiGetByExampleId(): Cypress.Chainable<unknown> {
```

Comment:
This naming of function supposes that the first argument is `exampleId` :think:

Maybe we should rename them to `tuiEnsureDocExample()`, `tuiGetDocExample()`, `tuiFindDocExample()` ?

Reply:
fixed